service/create/pki.go

```diff
@@ -0,0 +1,119 @@
+package create
+
+import (
+	"strings"
+
+	microerror "github.com/giantswarm/microkit/error"
+
+	"github.com/giantswarm/certctl/service/pki"
+	"github.com/giantswarm/certctl/service/token"
+)
+
+const (
+	// TODO Choose a sensible value and move to ConfigMap for V1
+	CertificateAuthorityTTL = "1440h"
```

Comment:
The CA TTL needs to be longer than the Cert TTL. For v1 is it OK to just use a config map? When we handle checking for certs expiring I think we'll also need to handle the CA expiring.